packages/contracts/src/2.0.0/protocol/Exchange/libs/LibMath.sol

```diff
@@ -239,7 +239,6 @@ contract LibMath is
         }
         // Compute remainder as before
         uint256 remainder = mulmod(target, numerator, denominator);
-        // TODO: safeMod
```

Comment:
Do you know what `safeMod` was going to check for?
Presumably just that the LHS argument is >= 0. In which case I guess we should be safe with only `safeSub`.

Reply:
I'm actually not sure, but it should be safe with the `denominator > 0` check.